pyrokinetics/gk_code/GKOutputReaderGENE.py

```diff
@@ -339,10 +366,15 @@ def _set_fluxes(
                     ]
 
                 # Skip time/data values in field print out is less
-                if i_time != data.ntime - 1:
+                if i_time < data.ntime - 2: 
```

Comment:
Isn't this going up to the second to last point? Is this because GENE always spits out final point regardless of time?

Reply:
Yes, GENE spits out the final step regardless, even if it is not integer multiples of istep_field or istep_nrg.